client/webserver/site/src/html/bodybuilder.tmpl

```diff
@@ -257,7 +257,7 @@
   </div>
 </div>
 
-<script src="/js/entry.js?v={{commitHash}}"></script>
+<script src="/js/entry.js"></script>
```

Comment:
I think this needed in order to avoid the browser returning a previous version of the js that was cached.

Reply:
Ok .. I am not great with js .. but the variable `commitHash` has been removed. I did test quite a bit after maing the above change.

Is this `?v={{commitHash}}` used as like an Id or a marker?

Reply:
And maybe the build itself can now create the `entry.js`, `entry.js.map`?

Reply:
Now using Webpack Build ID in place of `commitHash`